docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -59,10 +59,10 @@ You can either run it on a specific file, folder, or your entire codebase when c
 
 ```bash
 // Data Grid specific
-npx @mui/x-codemod v7.0.0/data-grid/preset-safe <path>
+npx @mui/x-codemod@latest v7.0.0/data-grid/preset-safe <path>
```

Comment:
Context: https://github.com/mui/mui-x/pull/11521#discussion_r1436637173

Reply:
Nice. Shouldn't hurt and ensures that people will use the latest package every time. 👍 

Have you considered adding `<!-- #default-branch-switch -->` or `<!-- #default-tag-switch -->` in all of these cases as they would need to change during the pre-release period. 🤔

Reply:
> they would need to change during the pre-release period.

Right, when pre-release for v8 starts, for v8 codemods (or for file `migration-data-grid-v7.md`), we could need to do `npx @mui/x-codemod@next` but the v7 ones would work fine with `npx @mui/x-codemod@latest`, so maybe keeping the `<!-- #default-branch-switch -->` [here](https://github.com/mui/mui-x/blob/783d7edf567e1865a2b4292b4df06d352a1418e5/packages/x-codemod/README.md#L13) should be enough. Wdyt?

Reply:
Yup, fair point. There is no real need to switch the tag on old migration docs. 👌 
If you think that it makes sense, we can add it only on `README.md`. 👍